scripts/capture_xcresult_schema.py

```
@@ -0,0 +1,204 @@
+    return sorted((row[1], row[2]) for row in rows)
+
+
+def developer_tools(connection, tables):
+    """The producing Xcode, if this bundle records it. None means the table is
+    absent; [] means it exists and Apple left it empty, which is what every
+    bundle inspected on 26.2 does."""
+    if "DeveloperTools" not in tables:
+        return None
+    columns = [name for name, _ in tables["DeveloperTools"]]
+    rows = connection.execute("SELECT * FROM DeveloperTools").fetchall()
+    return [dict(zip(columns, row)) for row in rows]
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)